.github/workflows/push-image.yml

```diff
@@ -78,29 +78,30 @@ jobs:
       if: inputs.build_image
       uses: actions/checkout@v4
 
-    - name: "Populate folder with input files"
+    - name: "Populate 05-assistive-chatbot folder with input files"
       if: inputs.build_image && ${{ inputs.dockerfile_folder }} == '05-assistive-chatbot'
-      env:
-        GURU_CARDS_URL: https://docs.google.com/uc?export=download&id=${{ secrets.GURU_CARDS_URL_ID }}
       run: |
-        echo "Downloading from ${GURU_CARDS_URL}"
-        curl -L "${GURU_CARDS_URL}" > download.zip
-        unzip -o download.zip
-        rm download.zip
-        mv guru_cards_for_nava--Multi-benefit.json guru_cards_for_nava.json
-
-        cd ${{ inputs.dockerfile_folder }}
-        mv ../guru_cards_for_nava.json .
-
         # The DOT_ENV_FILE_CONTENTS contains API keys, like LITERAL_API_KEY and OPENAI_API_KEY
         # As such, make sure the built image is not publicly accessible
         echo "${{ secrets.DOT_ENV_FILE_CONTENTS }}" > .env
 
+    - name: "Populate 04-call-summaries folder with input files"
+      if: inputs.build_image && ${{ inputs.dockerfile_folder }} == '04-call-summaries'
+      run: |
+        # The DOT_ENV_FILE_CONTENTS_04 contains API keys, like LITERAL_API_KEY and OPENAI_API_KEY
+        # As such, make sure the built image is not publicly accessible
+        echo "${{ secrets.DOT_ENV_FILE_CONTENTS_04 }}" > .env
```

Comment:
@ccheng26 to add `DOT_ENV_FILE_CONTENTS_04` to https://github.com/navapbc/labs-gen-ai-experiments/settings/secrets/actions